board/sharp/common/lcd.c

```diff
@@ -49,10 +49,15 @@ void mxsfb_system_setup(void)
 	struct mxs_clkctrl_regs *xtal =
 		(struct mxs_clkctrl_regs *)MXS_CLKCTRL_BASE;
 	struct mxs_pwm_regs *pwm = (struct mxs_pwm_regs *)MXS_PWM_BASE;
+
+#ifndef CONFIG_BRAIN_2G
 	int i, j;
+	uint8_t ili9805_mac = 0;
+#else
+	int i;
+#endif
```

Comment:
(IMO) 特に理由がない限りは、 `#ifdef` の方が直感的かつ後で #elif defined を増やす際ラクなのでそちらを使うようにしてください。PR で提出された変更のうち、 `#ifndef` を使っている箇所はこれ以外も同様です。`#else` 句のないものは `#ifdef BRAIN_3G_4G` を使う感じで。
```suggestion
#ifdef CONFIG_BRAIN_2G
	int i;
#else
	int i, j;
	uint8_t ili9805_mac = 0;
#endif
```

Reply:
# ifdefに統一しました